packages/hub/node-tests/card-service-test.ts

```diff
@@ -2004,7 +2004,6 @@ describe('hub/card-service', function() {
           { type: 'cards', id: vanGogh.canonicalURL },
           { type: 'cards', id: mango.canonicalURL },
         ]);
-        expect(doc).to.not.have.property('included');
```

Comment:
This was a policy change. 

OcclusionRules don't distinguish between value and reference. A field is either in or out, and that doesn't depend on whether it's a value or reference. But this means that if we would have wanted the field to be present in upstream attributes, it's necessarily also present in upstream included when it's a reference.

Reply:
An alternative that would let us control `included` separately is to distinguish between when to include the reference in `relationships` vs when to actually include the related resource in `included`. Today those are the same thing and the occlusion rules don't let you say one or the other.